js/post/update_counts.js

```diff
@@ -1,8 +1,14 @@
+creditCountSpec = 0;
+creditCountMaj = 0;
+creditCountMin = 0;
+creditCount300and400 = {'Spec': 0, 'Maj': 0, 'Min': 0};
+
+
 /**
  * Updates number of completed courses in Specialist.
 **/
 function updateCompletedSpecCourses () {
-    'use-strict';
+    'use-strict'
```

Comment:
Ah right, thanks @Ian-Stewart-Binks. 
Just a question though, are you sure it's not "use strict"; (with the quotations)?

Every source I've seen seems to use this. Example:
http://www.w3schools.com/js/js_strict.asp
https://msdn.microsoft.com/en-us/library/ie/br230269%28v=vs.94%29.aspx